### PR TITLE
Added `USER_AGENTS_CACHE` setting.

### PR DESCRIPTION
**Pull request by caxap**

### CODE REVIEW

File: django_user_agents/utils.py [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+from django.conf import settings
+from django.core.cache import DEFAULT_CACHE_ALIAS
 from user_agents import parse
 
 
+try:
+    # This function has been deprecated since Django 1.7 in favor of caches.
+    from django.core.cache import get_cache
+except ImportError:
+    from django.core.cache import caches
```
Comment (selwin) [inline django_user_agents/utils.py]: Mind reversing the order of the imports so we don't trigger deprecation warnings in Django 1.7?

**Comment by selwin** [comment]
It seems like the tests fail on Python 3, please fix them before I merge this PR in :)

**Comment by caxap** [comment]
@selwin I've updated and rebased PR.

**Comment by caxap** [comment]
@selwin any updates on this?

**Comment by selwin** [comment]
Merged, thanks for the PR!

**Pull request merged**